gammapy/maps/axes.py

```diff
@@ -3073,47 +3113,48 @@ def pix_to_idx(self, pix, clip=False):
 
     @property
     def center(self):
-        """Center of the label axis"""
+        """Center of the label axis."""
         return self._labels
 
     @property
     def edges(self):
-        """Edges of the label axis"""
+        """Edges of the label axis."""
+        # TODO: Why not return self._labels here?
```

Comment:
```suggestion
```
because labels are centers

Reply:
Fair enough!

Reply:
Oh but then why are `edges_min` and `edges_max` returning `self._labels`?

Reply:
Hmm, not sure :)